front/pages/api/w/[wId]/webhook_sources/views/[viewId]/index.ts

```diff
@@ -115,4 +149,37 @@ async function handler(
   }
 }
 
+async function propagateSystemViewNameChange(
```

Comment:
What about editWebhookSourceViewsName(webhookSourceViewId, name) and check that the webhookSOurceViewId is the system view